dashboard/app/views/levels/show.html.haml

```diff
@@ -48,8 +48,5 @@
 
 = render partial: 'levels/admin'
 
--#
-  HACK: The respond_to? test below is to workaround a bug where a server
```

Comment:
I'm not sure why the original hack was needed.  Either way, `professional_learning_course?` is a column on `Script` so it should be present if `Script` exists.  Maybe there's some weird edge case when Dashboard is reloading that shows up in HoneyBadger, but I'm not sure we want to defend against this everywhere.